number_parser/parser.py

```diff
@@ -116,12 +114,30 @@ def number_builder(token_list):
 
 SENTENCE_SEPERATORS = [".", ","]
 
-
-def parse(input_string):
-    # Fails when two numbers have no SENTENCE_SEPERATORS or words between them
-    # eg) 'one two three' doesn't work but 'one,two,three' and 'one apple , two mango , three' works.
+def tokeniser(input_string):
     all_tokens = re.split(r'(\W)', input_string)
+    return all_tokens
+
+def parse_number(input_string):
+    if input_string.isnumeric():
+        return int(input_string)
```

Comment:
Yes, I did realize that it won't work with float numbers. But was planning to fix it in a later iteration once the support for decimals is added. The main ```parse``` function would need to handle the 'point' keyword and the ```number_builder``` logic would also change. Even more, modifications would be needed for negative numbers as ' - ' (minus sign) would need to be treated separately. I guess it's better to raise a separate issue for decimals and work on that?

Reply:
You are right, open a new ticket and it will be fixed in a future iteration. If you can, please, link to this comments (URL: https:\//github.com/arnavkapoor/number-parser/pull/9#discussion_r442400344).